### PR TITLE
Create rule for Eventhubs

### PR DESCRIPTION
**Pull request by philip94**
Created ruleset Eventhubs.com.xml

### CODE REVIEW

**Comment by philip94** [comment]
Merge pull request @J0WI @cschanaj.

**Comment by cschanaj** [comment]
@philip94 I am not a reviewer and I cannot merge your PR. Also, I am not sure that you can ping for merge request here.

Just a few suggestions of mine:

- Remove empty comment

- Indent the `target` to the same column.

- Remove those unnecessary `test url` 

- More `target` can be added (you can refer to https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains)

**Comment by philip94** [comment]
@cschanaj Thank you for your suggestions. I provided one additional target, but the other do not seem to be functional, so I put them in the comments on top.

File: src/chrome/content/rules/Eventhubs.com.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		ngrtest.eventhubs.com
+		nmedia.eventhubs.com
+		smedia.eventhubs.com
+-->
+
+<ruleset name="Eventhubs">
+	
+	<target host="eventhubs.com" />
+	<target host="www.eventhubs.com" />
+    <target host="forums.eventhubs.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Eventhubs.com.xml]: Could you fix the code style here?

**Comment by jeremyn** [comment]
@philip94 Are you still interested in working on this pull request?

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @philip94 . I've noted `eventhubs.com` at `https://github.com/EFForg/https-everywhere/issues/3069#issuecomment-354338635` in case someone else wants to make a pull request for this domain.

**Pull request closed without merging**